pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "cloudcasting"
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
fine to leave this version bump in, but this hiding in the diffs has motivated me to see if we can find a better approach!